package.json

```diff
@@ -26,8 +26,8 @@
     "uninstall-local": "pip uninstall dash-core-components -y",
     "build:js": "webpack --mode production",
     "build:js-dev": "webpack --mode development",
-    "build:py": "node ./extract-meta src/components > dash_core_components/metadata.json && cp package.json dash_core_components && dash-generate-components ./src/components dash_core_components",
-    "build:r": "node ./extract-meta src/components > dash_core_components/metadata.json && cp package.json dash_core_components && dash-generate-components ./src/components dash_core_components --r-prefix 'core'",
+    "build:py": "dash-generate-components ./src/components dash_core_components",
+    "build:r": "dash-generate-components ./src/components dash_core_components --r-prefix 'dcc'",    
```

Comment:
Again, let's 🔪 `extract-meta`

Reply:
fixed in 2a7808473d90376f8487e18acb2ed12d429463c8